### DISCUSSION TITLE
After Plex login, getting message "rawlist argument choices cannot exceed length of 9"

### DISCUSSION THREAD

**Discussion by rich-r** [Taxel/PlexTraktSync#1390] [Support] [answered]
Using version 0.25.12 on unRAID via Community Apps r
Repo: ghcr.io/taxel/plextraktsync:latest

In the container console, I enter 'python3 -m plextraktsync sync', sign in with Plex, and get the following result:

Login to MyPlex was successful
[...]
Select default server: 1Traceback (most recent call last):
[...]
InquirerPy.exceptions.InvalidArgument: rawlist argument choices cannot exceed length of 9

Full log attached. Anyone know what I'm doing wrong?
[log.log](https://github.com/Taxel/PlexTraktSync/files/10476085/log.log)

**Comment by braydin72** [comment]
Mine is also doing this.

**Reply by rich-r** [reply]
Just came back to this and noticed unRAID is not officially supported. I set it up instead on a Raspberry pi and got past this stage. 

FWIW the _select default server_ dialog gave me 10 choices (some of my media players came up in the owned servers list)

**Comment by glensc** [comment, marked as answer]
Fixed long time ago:
- https://github.com/Taxel/PlexTraktSync/pull/1398
